src/social/components/Sting/StingModal.tsx

```diff
@@ -52,10 +54,13 @@ export const StingModal = ({ userAccessCode, pathToContent, isOpen, onClose }: S
     reset();
   }, [isOpen]);
 
+  const { user }: { user: any } = useUser(userAccessCode);
```

Comment:
Can you use `ReturnType<typeof useUser>` here?

Reply:
Doesn't work because it knows about `user`, but not what's inside `user`
<img width="251" alt="image" src="https://github.com/noom/community-web-uikit/assets/10712840/40cd69b7-3876-4fff-b02e-e32eb00e1cf2">